src/imitation/algorithms/adversarial/common.py

```diff
@@ -451,6 +454,12 @@ def _get_log_policy_act_prob(
             )
         elif isinstance(self.policy, sac_policies.SACPolicy):
             gen_algo_actor = self.policy.actor
+            if gen_algo_actor is None:
```

Comment:
I think it can only happen if `SACPolicy._build` hasn't been called yet: https://github.com/DLR-RM/stable-baselines3/blob/master/stable_baselines3/sac/policies.py#L292 But that should never happen as `SACPolicy.__init__` always calls `self._build` at the end: https://github.com/DLR-RM/stable-baselines3/blob/master/stable_baselines3/sac/policies.py#L289

So I think this is just a false positive from your type checker? I'd add an `assert gen_algo_actor is not None` to quieten it and move on.